.github/workflows/CI.yml

```diff
@@ -0,0 +1,43 @@
+name: wok-CI
+on: ["push"]
+
+jobs:
+  build-and-test-wok:
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: checkout
+        uses: actions/checkout@v1
+
+      - name: install pyyaml
+        run: |
+          sudo apt install python3-setuptools python3-dev
+          pip3 install pyyaml 
+
+      - name: update apt
+        run: |
+          sudo apt update 
+
+      - name: install distro depedencies
+        run: |
+          sudo apt install -y $(python3 -c 'import yaml;print(" ".join(yaml.load(open("dependencies.yaml"), Loader=yaml.FullLoader)["runtime-deps"]["ubuntu"]).replace("python3-cheetah", ""))')
```

Comment:
Why don't install python3-cheetah?

Reply:
Got errors:

https://github.com/ramonmedeiros/wok/commit/a8030716edb8d8dac4b8b72e5eb8c7f35621015a/checks?check_suite_id=347241992

```E: Package 'python3-cheetah' has no installation candidate```

Reply:
what OS is that?

Reply:
ubuntu 18.04